src/main/java/io/skai/okta/internshipstreamingserviceparserxml/services/EpisodesConverter.java

```diff
@@ -0,0 +1,17 @@
+package io.skai.okta.internshipstreamingserviceparserxml.services;
+
+import io.skai.okta.internshipstreamingserviceparserxml.dto.Episode;
+
+import java.time.LocalDateTime;
+
+public interface EpisodesConverter {
```

Comment:
what is the purpose of this converter?

Reply:
If we will use other dto with other field names, or other format of publishing date, we just need new implementation. 
For example imdb use other "title", "id", "description". So we will create new episodeConverterImpl and new episodeRepositoryImpl with this converter.
Or i need remove this interface?

Reply:
How you will implement other converter for other DTO if all methods of this interface working with Episode? 

The proper way of creating such abstraction is to bring back RssItem which was used in LostfilmParser and then in service convert RssItem to Episode

Reply:
yes, can i do that? just for training.

Reply:
sure